tests/chainer_tests/test_link.py

```diff
@@ -468,6 +470,12 @@ def test_update_enabled(self):
         self.link.enable_update()
         self.assertTrue(self.link.update_enabled)
 
+    def test_count_params(self):
+        self.assertEqual(self.link.count_params(), 8)
+        with warnings.catch_warnings(record=True) as w:
```

Comment:
Check the number of warnings issued is correct (maybe 1) and the type of the warning is `UserWarning` with `w`. Otherwise, remove `as w` as it is not used.